.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   // TODO: use the new `eslint-config-node`?
   extends: ['@commercetools-frontend/eslint-config-mc-app'],
   rules: {
+    'import/export': 'off', // To avoid the error: `RangeError: Maximum call stack size exceeded`
```

Comment:
Where is the `RangeError: Maximum call stack size exceeded` error coming from?

Reply:
It's from CI when running linters / tests. The CI workflow on PRs is configured with `--changedSince=main`, guessing due to the amount of sample data files to process.

Once we decouple the sample data, we can re-adjust some of those things.

Reply:
Ah! If I understand correctly, it's because this PR has lots of files changed but it can be reverted afterwards, right?
Can we leave a TODO comment so we don't forget about it? 🙏

Reply:
Yep https://github.com/commercetools/test-data/pull/801/commits/ce54ae76758af750c9d379c3d49de6aab4750551